### PR TITLE
Removing all logic of sql bindings

### PR DESCRIPTION
**Pull request by VasuBhog**
Remove all sql-bindings from vscode-mssql as it is now in the sql-bindings repo. 
This will be the first step to remove and then bundle the sql-bindings extension in the extension pack.

### CODE REVIEW

File: typings/sql-bindings.d.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+declare module 'sql-bindings' {
```
Comment (Charles-Gagnon) [inline typings/sql-bindings.d.ts:7]: Why is this here? The extension shouldn't need to have this.

Comment (VasuBhog) [reply typings/sql-bindings.d.ts:7]: Yeah, I can delete the typing files as well now.

**Review by Charles-Gagnon — APPROVED** [review]
Make sure you go through the package dependencies and see if there's any of those we can remove as well

**Comment by VasuBhog** [comment]
> Make sure you go through the package dependencies and see if there's any of those we can remove as well

Checked and found there are no package dependencies that we can remove.
